modules/apps/27-interchain-accounts/host/keeper/keeper.go

```diff
@@ -199,3 +205,33 @@ func (k Keeper) SetInterchainAccountAddress(ctx sdk.Context, connectionID, portI
 	store := ctx.KVStore(k.storeKey)
 	store.Set(icatypes.KeyOwnerAccount(portID, connectionID), []byte(address))
 }
+
+// GetAuthority returns the 27-interchain-accounts host submodule's authority.
+func (k Keeper) GetAuthority() string {
+	return k.authority
+}
+
+// GetParams returns the total set of the host submodule parameters.
+func (k Keeper) GetParams(ctx sdk.Context) types.Params {
+	store := ctx.KVStore(k.storeKey)
+	bz := store.Get([]byte(types.ParamsKey))
+	if bz == nil {
+		panic("ica/host params have not been set")
+	}
+
```

Comment:
Afaik we normally check `len(bz) == 0`

If `len(bz) == 0` then should we be panicking or return `types.DefaultParams()`?